source/type/pb_converter.h

```diff
@@ -440,13 +440,14 @@ struct PbCreator {
         }
 
         template <typename NT, typename PB>
-        void fill_comments(const NT* nt, PB* pb, const std::string& type = "standard") {
+        void fill_comments(const NT* nt, PB* pb) {
             if (nt == nullptr) {
                 return;
             }
             for (const auto& comment : pb_creator.data->pt_data->comments.get(nt)) {
                 auto com = pb->add_comments();
-                com->set_value(comment);
+                auto type = comment.type.empty() ? "standard" : comment.type;
```

Comment:
```suggestion
                auto type = comment.type.empty() ? "information" : comment.type;
```
maybe ?

Reply:
I choose `standard` to prevent changes when `comment_type` isn't set, but I would prefer information, if no one object I will change it